brightlayer-ui/brightlayer-ui-cli
# Feature/pxblue 2223 update meta

Fixes # .


#### Changes proposed in this Pull Request:
- add keyword section to package.json with keywords to display on NPM
- point to master branch for cli templates
- 


#### Screenshots:


#### To Test:
- verify keyword section
- verify item 5 in root readme points to master branch for all cli template references
